tiledb/common/logger.h

```diff
@@ -36,6 +36,10 @@
 #define TILEDB_LOGGER_H
 // clang-format off
 // Order here matters to avoid linker errors
+#if defined(__MINGW32__)
+#define USING_R
+#define R_R_H
+#endif
```

Comment:
I think this is due to my package RcppSpdlog and its needed to R i/o streams.  It may not be needed for mingw builds of the library itself.